Makefile

```diff
@@ -20,4 +20,4 @@ lint:
 
 ## Perform all tests
 test:
-	go test $(go list ./... | grep -v vendor)
+	go test ./...
```

Comment:
This changed in a recent version, from `go help packages`:
> any slash-separated pattern element containing a wildcard never participates in a match of the "vendor" element in the path of a vendored package, so that ./... does not match packages in subdirectories of ./vendor or ./mycode/vendor